src/scripting_api/aformat.js

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2020 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+class AFormat {
```

Comment:
I was tired when I changed that...
So the "correct" name is `AForm` because all theses functions lived in a file (distributed with Acrobat few years ago) called `AForm.js` (see http://www.sfu.ca/~wcs/ForGraham/Aladdin%20stuff/Acrobat%20Reader%205.0/Contents/MacOS/JavaScripts/AForm.js).
There's no real documentation about those: only few indications here https://www.adobe.com/content/dam/acom/en/devnet/acrobat/pdfs/FormsAPIReference.pdf#page=119.
In pdfium, these functions are implemented in a file called `cjs_publicmethods.cpp`.
So I guess the specifications are the code itself as found in `AForm.js`.
I implemented them here:
https://github.com/calixteman/pdf.js/blob/sandbox_experimental/src/scripting_api/aformat.js
but it's for an other PR.